docs/glossary.rst

```diff
@@ -71,16 +87,17 @@ Glossary
 
    Github
       GitHub is an online platform where one can store and share version controlled projects
-      using Git (and thus also DataLad project).
+      using Git (and thus also DataLad project).See
+      `Github.com <https://github.com/>`_.
 
    Gitk
       A repository browser that displays changes in a repository or a selected set of commits. It
       visualizes a commit graph, information related to each commit, and the files in the trees
       of each revision.
 
-   Github
-      An online platform to host and share software projects version controlled with :term:`Git`. See
-      https://github.com/.
+   Gitlab
```

Comment:
Hm. Actually it's a software to run such a platform. But I'm not sure, whether that distinction should be made here. May be it would just be confusing.